package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ansible-vault",
-  "version": "1.0.3",
+  "version": "1.0.4",
```

Comment:
can you please change back to 1.0.3 or move to 1.0.4-0.
I am using npm version to tag the package.

Reply:
No problem. Reverted this file to 1.0.3.